02.create-and-explore/06.all-about-scripting/03.avatar-scripts/docs.md

```diff
@@ -28,9 +28,7 @@ In order to attach a script to your avatar, you must modify your avatar's FST fi
 When packaging your model, the tool will iterate through the directory and add each script to the FST file. Now, you can upload the folder structure to the Marketplace or to your preffered hosting service.
 ## How to Stop an Avatar Script
  You can temporarily disable or restart your avatar script in the script interface (crtl + J) by pressing the close or reload button. 
- ![](\endFlow.png)
+ ![](endFlow.png)
```

Comment:
We don't need to show the entire image, only the part of the dialog that's useful. 
![image](https://user-images.githubusercontent.com/42979611/49473025-c6c1ae80-f7c5-11e8-9645-de77ffc0e6a8.png)